src/main/resources/Macros/KanbanAWMSource.xml

```diff
@@ -140,7 +140,7 @@
   #foreach($currentpage in $results)
     #set($currentdoc = $xwiki.getDocument($currentpage))
     #set($value = $currentdoc.getValue($category))
-    #if(!$categories.contains($value) &amp;&amp; (!$selectedCategories || $selectedCategories.contains($value)))
+    #if(!$categories.contains($value) &amp;&amp; $value &amp;&amp; (!$selectedCategories || $selectedCategories.contains($value)))
```

Comment:
If `$value` is null, then the first check is not relevant
So I would change the order to `$value && !$categories.contain...`